src/main/scala/org/phenoscape/kb/CharacterDescription.scala

```diff
@@ -239,13 +241,13 @@ object AnnotatedCharacterDescription { //FIXME
           result.getLiteral("description").getLexicalForm,
           CharacterMatrix(
             IRI.create(result.getResource("matrix").getURI),
-            result.getLiteral("matrix_label").getLexicalForm)),
+            result.getLiteral("matrix_label").getLexicalForm), MinimalTerm(IRI.create(result.getResource("character").getURI), result.getLiteral("characterLabel").getLexicalForm)),
```

Comment:
@Shalsh23 if you look at the places in the code that call this function, you can see that none of the query results have `character` or `characterLabel`. So this will always cause a null pointer exception. This is the cause of #172.